courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<p>We can also add a TensorBoard() callback to inspect the model in TB (<a href="https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard" target="_blank">https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard</a>)</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Done.